src/bgw/README

```diff
@@ -0,0 +1,34 @@
+TimescaleDB needas to run multiple background jobs. This module
+adds a simple scheduler so that jobs inserted into a jobs table
+could be run on a schedule. This first implementation has two limitations:
```

Comment:
We need to describe the kinds of schedules we can run. At minimum we can run a job a fixed interval after the last time the job completed. Can we also run a job at a fixed rate?

Reply:
added a section